contrib/node/examples/src/node/hello-test/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
When pants runs this target, it will do so under `.pants.d`... so there shouldn't be a node_modules folder created here.